uint8 GRASP=0
uint8 PLACE=1
uint8 OPEN_GRIPPER=2
uint8 CLOSE_GRIPPER=3
uint8 MOVE_ARM=4
uint8 RAISE_ARM=5
uint8 LOWER_ARM=6
uint8 RESET_ARM=7

uint8 action_type
string object
geometry_msgs/Point position